seed/management/commands/delete_unused_uploaded_files.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+"""
+:copyright (c) 2014 - 2022, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from seed.data_importer.models import ImportFile
+
+import os
+
+
+class Command(BaseCommand):
+    help = 'Creates a default super user for the system tied to an organization'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--org_id',
+                            default=None,
+                            help='Name of a specific organization to operate',
+                            action='store',
+                            dest='org_id')
+
+    def handle(self, *args, **options):
+        # In local dev and production the files are in the media/uploads folder.
+        # Folder has: 4491 files
+        # Database has: 7306 files
+        #
+        # Steps to process
+        #   1. The database has various paths of all the files as we have been
+        #      moving the files around and the database paths have not been updated
+        #      First task is to update the file paths to /seed/media/uploads for the files
+        #   2. There is a deleted column in SEED that is updated when a user removes the
+        #      file from the frontend; however, SEED persists the file. This tasks
+        #      deletes the files from disk (if they exist), but it does not delete the
+        #      ImportFile record itself
+
+        org_id = options['org_id']
+        if org_id:
+            files = ImportFile.objects.get_all().filter(import_record__super_organization=org_id)  # this actually returns a queryset
+        else:
+            files = ImportFile.objects.get_all()
+
+        # fix the file path if it is not /seed/media/uploads
+        # populate the list of files - this may need to be broken up into mulitple tasks on the real data
+        # based on how large the directory / table is.
+        rename_files = []
+        for f in files:
+            filename = f.file.name
+            if os.path.exists(filename):
+                # don't do anything if the path exists
+                continue
+
+            # the filename/path is not correct and needs to be updated
+            # put it in a tuple of db object, oldname, new name
+            new_base_path = "/seed/media/uploads"
+            if filename.startswith(new_base_path):
```

Comment:
Won't this case for fixing the upload path get skipped if we `continue` here?

Reply:
Also I just noticed the comment above about updating the filepath/path isn't totally true - we don't seem to be doing that here. Am I misunderstanding?

Reply:
if the file path is formed correctly or empty then it will skip that file, but continue to the next file. Maybe this is being confused with `break`?